rust/Cargo.toml

```diff
@@ -1,15 +1,24 @@
+[workspace]
+members = ["driver"]
+
 [package]
 name = "evmrs"
 version = "0.1.0"
 edition = "2021"
 
+[features]
+# workaround for enabling mock feature also in integration tests
```

Comment:
This comment could be formulated in a more descriptive way, for non-versed people to understand better
```suggestion
# Flag mock enables code generation for test mocking. This allows generating mocks for other build configuration but "test". 
```